src/pyne/remove_unsupported.py

```diff
@@ -1,8 +1,16 @@
-with open('pyne.cpp', 'r', encoding='utf-8') as reader:
-    lines = reader.readlines()
+import sys
 
-writer = open('pyne.cpp.new', 'w', encoding='utf-8')
+if sys.version > '3':
```

Comment:
is this because windows is stuck with Py 2?

Reply:
I don't know, but with the new change I PRed this shouldn't be necessary....

Reply:
should we try without? or is this necessary for some users stuck on Py2 systems?